src/testController.ts

```diff
@@ -153,7 +153,15 @@ export class TestController {
     await this.telemetry.sendCodeLensEvent("test_in_terminal");
 
     if (this.terminal === undefined) {
-      this.terminal = vscode.window.createTerminal({ name: "Run test" });
+      const previousTerminal = vscode.window.terminals.find(
+        (terminal) => terminal.name === "Ruby LSP: Run test"
```

Comment:
Nit: perhaps we can store `"Ruby LSP: Run test"` in a variable to prevent this and the below value out-of-sync?